src/Evolution/Executables/GeneralizedHarmonic/GeneralizedHarmonicBase.hpp

```diff
@@ -223,6 +226,8 @@ struct ObserverTags {
                       volume_dim, Frame::ElementLogical, Frame::Inertial>,
                   ::domain::Tags::Mesh<volume_dim>>,
               gr::Tags::WeylElectricCompute<DataVector, 3, Frame::Inertial>,
+              gr::Tags::WeylTypeD1Compute<DataVector, 3, Frame::Inertial>,
```

Comment:
Did you want to add `WeylElectricScalarCompute` here as well?

Reply:
Missed that, thanks!